src/cluster.c

```diff
@@ -7011,6 +7011,7 @@ unsigned int delKeysInSlot(unsigned int hashslot) {
         de = dictEntryNextInSlot(de);
         robj *key = createStringObject(sdskey, sdslen(sdskey));
         dbDelete(&server.db[0], key);
+	propagateDeletion(&server.db[0], key, server.lazyfree_lazy_server_del);
```

Comment:
Here we may need to notify that the key is invalid for tracking clients

Reply:
server.dirty also should be add 1

Reply:
maybe also key space notification to modules? @MeirShpilraien WDYT?

Reply:
Agree, if key was delete we should should a notification about it.

Reply:
I suppose signalModifiedKey as well.